Source/cielim/Private/Spacecraft.cpp

```diff
@@ -0,0 +1,39 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "Spacecraft.h"
+
+// Sets default values
+ASpacecraft::ASpacecraft()
+{
+ 	// Set this actor to call Tick() every frame.  You can turn this off to improve performance if you don't need it.
+	PrimaryActorTick.bCanEverTick = true;
+
```

Comment:
:small_red_triangle_down: I'll not continue writing it after this instance. You can remove the redundant new line all throughout.

Reply:
Fixed